.github/workflows/lint.yml

```
@@ -6,17 +6,20 @@ on:
   pull_request:
 
 jobs:
-  styles:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v4
-    - name: Install PHP
-      uses: shivammathur/setup-php@v2
-      with:
-        php-version: '8.2'
 
-    - name: Run Script
-      run: testing/run_cs_check.sh
+  style:
+    name: PHP Style Check
+    uses: GoogleCloudPlatform/php-tools/.github/workflows/code-standards.yml@add-rules-for-cs-fixer
```

change to `main` once https://github.com/GoogleCloudPlatform/php-tools/pull/152 is merged
```suggestion
    uses: GoogleCloudPlatform/php-tools/.github/workflows/code-standards.yml@main
```